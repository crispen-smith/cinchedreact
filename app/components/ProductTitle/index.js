/**
 *
 * ProductTitle
 *
 */

import React from 'react';
import propTypes from 'prop-types';
import styled from 'styled-components';

const PT = styled.h1`
  margin: 1rem 0.5rem 0 0.5rem;
  padding: 0;
  text-align: center;
  font-size: 3rem;
  font-family: 'Monsieur La Doulaise', cursive;
`;

const ProductTitle = props => <PT>{props.children}</PT>;

ProductTitle.propTypes = {
  children: propTypes.oneOfType([
    propTypes.arrayOf(propTypes.element),
    propTypes.element,
    propTypes.string,
    propTypes.node,
  ]),
};

export default ProductTitle;
