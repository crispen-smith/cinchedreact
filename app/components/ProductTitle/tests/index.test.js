import React from 'react';
import renderer from 'react-test-renderer';

import ProductTitle from '../index';

describe('<ProductTitle />', () => {
  it('Renders a ProductTitle', () => {
    const tree = renderer.create(<ProductTitle>TEST</ProductTitle>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
