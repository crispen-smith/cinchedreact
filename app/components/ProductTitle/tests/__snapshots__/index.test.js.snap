// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductTitle /> Renders a ProductTitle 1`] = `
<h1
  className="ProductTitle__PT-iIqmTV bAjZGN"
>
  TEST
</h1>
`;
